Sources/MapboxMaps/Foundation/MapboxMap.swift

```diff
@@ -178,6 +178,25 @@ public final class MapboxMap: MapboxMapProtocol {
         __map.setRenderCacheOptionsFor(cacheOptions)
     }
 
+    /// Sets whether multiple copies of the world will be rendered side by side beyond -180 and 180 degrees longitude.
+    ///
+    /// If disabled, when the map is zoomed out far enough that a single representation of the world does not fill the map's entire container,
+    /// there will be blank space beyond 180 and -180 degrees longitude.
+    /// In this case, features that cross 180 and -180 degrees longitude will be cut in two (with one portion on the right edge of the map
+    /// and the other on the left edge of the map) at every zoom level.
+    ///
+    /// By default, renderWorldCopies is set to `true`.
+    ///
+    /// - Parameter isEnabled: The boolean value defining whether rendering world copies is going to be enabled or not.
+    public func setRenderWorldCopies(_ isEnabled: Bool) {
+        __map.setRenderWorldCopiesForRenderWorldCopies(isEnabled)
+    }
+
+    /// Returns whether multiple copies of the world are being rendered side by side beyond -180 and 180 degrees longitude.
+    public func getRenderWorldCopies() -> Bool {
```

Comment:
IMO this would look better as a property: `var renderWorldCopies: Bool`/`var shouldRenderWorldCopies`. Should it we expose it this way instead?

Reply:
We even have a precedent here: https://github.com/mapbox/mapbox-maps-ios/blob/49594a8a1b822c89d9022bbdf6425512c8c07726/Sources/MapboxMaps/Foundation/MapboxMap.swift#L139-L146

Reply:
I thought about that but I want to align the API with Android, which has both get and set methods.

Reply:
My +1 for having a variable instead of methods. Not quite sure how much we should align APIs. This one looks exactly like a variable with a custom getter-setter.

Reply:
@alexander-kulikovskii @yunikkk Did you have the same option in Android? If so, why did you choose separate functions?

Reply:
@OdNairy it's because we override these methods, declared in Java. Additionally for consistency reasons, I suppose.